web/packages/plugins/dmt-app/src/pages/editor/data-source/AddDatasource.tsx

```diff
@@ -25,6 +25,7 @@ const constructType = (selectedDatasourceType: string) => {
 }
 
 const AddDataSourceComponent = () => {
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore